cea/datamanagement/weather_helper/weather_helper.py

```
@@ -74,6 +74,31 @@ def copy_weather_file(source_weather_file, locator):
+
+        # from dat to epw
+        with open(output_epw_file, 'w') as epw_file:
+            # add EPW-Header
+            epw_file.write("LOCATION,Example,DEU,,,0,0,0\n")
+            epw_file.write("DESIGN CONDITIONS,0\n")
+
+            for _, row in weather_data.iterrows():
+                epw_line = f"{row['Year']},{row['Month']},{row['Day']},{row['Hour']},...,{row['Temperature']},...\n"
+                epw_file.write(epw_line)
+    except Exception as e:
+        raise ValueError(f"Error parsing .dat-Datei: {e}")
```

Ruff (0.8.0)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)